core/src/UserSettingsDialog.html

```diff
@@ -370,87 +384,8 @@
     background-color: var(--sapBackgroundColor, #f7f7f7);
   }
 
-  .lui-usersettings-burger {
-    margin-right: 16px;
-    cursor: pointer;
-    display: none;
-  }
-
-  .fd-nested-list__item {
-    display: flex;
-    -webkit-box-align: center;
-    align-items: center;
-    position: relative;
-    -webkit-box-flex: 1;
-    flex: 1 1 auto;
-    max-width: 100%;
-    border-bottom-color: var(--sapList_GroupHeaderBorderColor, #d9d9d9);
-  }
-
-  .lui-usersettings-list-item__info-container {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    display: flex;
-    align-items: center;
-  }
-
-  .fd-nested-list__link {
-    height: 5rem;
-    padding: 1rem;
-  }
-
-  .lui-usersettings-list-item__info-container .sap-icon {
-    font-size: 2rem;
-    color: var(--sapContent_NonInteractiveIconColor);
-    margin-right: 0.75rem;
-  }
-
-  .fd-nested-list__icon img {
-    max-width: 2rem;
-    max-height: 2rem;
-    border-radius: 0.25rem;
-    margin-right: 0.75rem;
-  }
-
-  .lui-usersettings-list-item__title-container {
-    display: flex;
-    flex-direction: column;
-    flex: auto;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-
-  .lui-usersettings-list-item__title {
-    color: var(--sapList_TextColor);
-    font-size: var(--sapFontSize, 14px);
-    font-weight: normal;
-    padding-top: 0.125rem;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    flex: 1 1 auto;
-  }
-
-  .fd-nested-list__link:active .lui-usersettings-list-item__title,
-  .fd-nested-list__link:active .lui-usersettings-list-item__subtitle,
-  .fd-nested-list__link:active .fd-nested-list__icon {
-    color: var(--sapList_Active_TextColor, #fff);
-  }
-
-  .lui-usersettings-list-item__subtitle {
-    font-size: var(--sapFontSize, 14px);
-    font-weight: normal;
-    color: var(--sapContent_LabelColor);
-    padding-top: 0.5rem;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    flex: 1 1 auto;
-  }
-
   .iframeUserSettingsCtn {
+    display: none;
```

Comment:
I don't know where this selector _"iframeUserSettingsCtn"_ is later used in detail, but wouldn't a **display:none** make the other properties obsolete. I can see that in DevTools this line is marked with a ~strikethrough~. I'm just curious if there's something behind this that I don't know.

Reply:
thanks, leftovers from removing the scroll on load. Removed